static/js/lib/courseAPI.js

```diff
@@ -0,0 +1,18 @@
+import moment from "moment"
+import { complement, isNil } from "ramda"
+import { RunEnrollment } from "../flow/courseTypes"
+
+export const notNil = complement(isNil)
```

Comment:
`notNil` already exists in the codebase (`static/js/lib/util.js` line 52). You can import it instead of defining it here